templates/al2/provisioners/install-worker.sh

```diff
@@ -272,9 +272,30 @@ elif [ "$BINARY_BUCKET_REGION" = "eu-isoe-west-1" ]; then
 elif [ "$BINARY_BUCKET_REGION" = "us-isof-south-1" ]; then
   S3_DOMAIN="csp.hci.ic.gov"
 fi
+
+# TODO: start deprecating these.
 S3_URL_BASE="https://$BINARY_BUCKET_NAME.s3.$BINARY_BUCKET_REGION.$S3_DOMAIN/$KUBERNETES_VERSION/$KUBERNETES_BUILD_DATE/bin/linux/$ARCH"
 S3_PATH="s3://$BINARY_BUCKET_NAME/$KUBERNETES_VERSION/$KUBERNETES_BUILD_DATE/bin/linux/$ARCH"
 
+function s3_binary_path() {
+  local binary_name=""
+  local os="linux"
+  local arch="$ARCH"
+  local version=""
```

Comment:
how could we support a version marker for binaries? e.g.

```
bin/$binary_name/latest.txt => 1.2.3

bin/$binary_name/1.2.3/$os/$arch/$binary_name
```

Reply:
i'm on board with this marker, but I don't think i'm going to make any change in this PR to facilitate any kind of `latest` or `*` pointer yet. it'll suffice for now to allow populating `cni_plugin_version` manually with the value of `$(curl https://amazon-eks.s3.us-west-2.amazonaws.com/cni-plugins/latest.txt)`